js_modules/dagit/packages/core/src/ui/Menu.tsx

```diff
@@ -127,3 +158,39 @@ const StyledMenuItem = styled(MenuItem)<StyledMenuItemProps>`
     outline: none;
   }
 `;
+
+const StyledMenuLink = styled(Link)`
+  text-decoration: none;
```

Comment:
Would it be possible to pull the styles shared with `MenuItemWIP` out into a `styled.css` fragment and then re-use them here? I'm not sure how styled-components does this but it seems like in traditional CSS we'd want to say `.MenuItemWIP, .StyledMenuLink {` on the same declaration to make sure that future changes are made to both.